cmd/amp/function.go

```diff
@@ -42,6 +46,7 @@ var (
 	removeFunctionCmd = &cobra.Command{
 		Use:   "rm FUNCTION",
```

Comment:
FUNCTION should really be something `FUNC-NAME-OR-ID` but I don't think we support name right now (or even ID fragments, like the other commands. To make it clear, we should probably say FUNC-ID for now.